### DISCUSSION TITLE
Correspondence between lockfileVersion and pnpm version

### DISCUSSION THREAD

**Discussion by KyrieLii** [pnpm/pnpm#7481] [Q&A]
Is there any documentation explaining the relationship between the two version numbers? Thanks !

**Comment by zkochan** [comment]
Yes, here: https://github.com/pnpm/spec/tree/master/lockfile#readme

## Relation of pnpm version to lockfile version

| lockfile version | used by pnpm versions |
| --   | --             |
| 6.0  | >=8.0.0
| 5.4  | >=7.0.0 <8.0.0 |
| 5.3  | >=6.0.0       |
| 5.2  | >=5.10.0       |
| 5.1  | >=3.5.0       |
| 5.0  | >=3.0.0       |
| 3.9 and 4.0  | >=2.17.0       |
| 3.9  | >=2.13.3       |
| 3.8  | >=2.8.0       |
| 3.7  | >=2.0.0       |
| 3.6  | >=1.43.0       |
| 3.5  | >=1.40.0       |
| 3.4  | >=1.23.0       |
| 3.3  | >=1.22.0       |
| 3.2  | >=1.18.1       |
| 3.1  | >=1.17 <1.18.1 |
| 3    | >=1 <1.17      |
| 2    | >=0.62 <1      |
